internal/operators/mce/manifest.go

```diff
@@ -34,6 +48,24 @@ func Manifests() (openshiftManifests map[string][]byte, customManifests []byte,
 	return openshiftManifests, mceManifest, nil
 }
 
+func getStorageSize(storageName string) string {
+	if intSize, ok := storageSizeGi[storageName]; ok {
+		return fmt.Sprintf("%dGi", intSize)
+	}
+	return ""
+}
+
+func GetAgentServiceConfigWithPVCManifest(storageClassName string) ([]byte, error) {
+	vars := map[string]string{
+		"DATABASE_STORAGE_SIZE":   getStorageSize(databaseStorage),
+		"FILESYSTEM_STORAGE_SIZE": getStorageSize(filesystemStorage),
+		"IMAGE_STORAGE_SIZE":      getStorageSize(imageStorage),
+		"STORAGE_CLASS_NAME":      storageClassName,
```

Comment:
Does creating the storage operator create a default storage class in the cluster?
I assume it would make things easier if you didn't have to provide this, right?

Reply:
It would definitely make it easier, but unfortunately it doesn't create a default storage class :(